datahub-web-react/src/app/entityV2/shared/tabs/Dataset/Schema/components/SchemaFieldDrawer/SchemaFieldDrawer.tsx

```diff
@@ -193,6 +193,8 @@ export default function SchemaFieldDrawer({
                 setSelectedTabName,
                 refetch,
                 refetchNotes,
+                fieldUrn: expandedField?.schemaFieldEntity?.urn,
```

Comment:
i'm not positive if this schemaField entity always exists? but maybe we pass it back from graphql even if it doesn't exist in the DB and we construct the urn and type etc. i'll check

Reply:
okay yeah i _think_ we always return a schema field urn even if it doesn't exist in the DB